src/Physics/Plugins/babylon.cannonJSPlugin.ts

```diff
@@ -375,9 +372,9 @@
                 var translation = mesh.getBoundingInfo().boundingBox.center.subtract(center).subtract(mesh.position).negate();
 
                 this._tmpPosition.copyFromFloats(translation.x, translation.y - mesh.getBoundingInfo().boundingBox.extendSize.y, translation.z);
-                //add it inverted to the delta 
-                this._tmpDeltaPosition.copyFrom(mesh.getBoundingInfo().boundingBox.center.subtract(c));
-                this._tmpDeltaPosition.y += mesh.getBoundingInfo().boundingBox.extendSize.y;
+                // comments needed - wingnut removes a .subtract(c) and then adjusts z
+                this._tmpDeltaPosition.copyFrom(mesh.getBoundingInfo().boundingBox.center);
+                this._tmpDeltaPosition.z = -mesh.getBoundingInfo().boundingBox.extendSize.z;

```

Comment:
Uh oh, I think I have more trouble.  I forgot to test non-square terrains.  
600 x 200 terrain fails with cannonJS error.
200 x 600 fails in my impostor positioning.
CRAP CRAP CRAP!  *sigh*
Back to the drawing board, I suppose.